gold-phone-input.html

```diff
@@ -157,9 +157,27 @@
       }
     },
 
-    observers: [
-      '_computeValue(value)'
-    ],
+    listeners: {
+      'input': '_onInput'
+    },
+
+    ready: function() {
+      this.addEventListener('blur', function() {
+        this._handleAutoValidate();
```

Comment:
I'm surprised that this works, given that the context is not correctly bound. Do we have a test that covers this behavior?